src/scripts/components/widgets/modals/Modal.js

```diff
@@ -20,7 +20,7 @@ const Wrapper = styled.div`
   position: fixed;
   top: 0;
   width: 100%;
-  z-index: 10;
+  z-index: 999;
```

Comment:
The staking modal did not have a high-enough z-index to render above all of our buttons

Reply:
thanks for fixing the z-index, @bent0b0x